.env.example

```diff
@@ -1,5 +0,0 @@
-REACT_APP_LUMINUS_CLIENT_ID=your_luminus_client_id_here
```

Comment:
Seems to me this mock .env file was deleted by accident. Could you please restore it back?

Reply:
Sure. I’ll change it.